src/t_hash.c

```diff
@@ -1845,10 +1848,11 @@ static ExpireAction hashTypeActiveExpire(eItem _hashObj, void *ctx) {
     ActiveExpireCtx *activeExpireCtx = (ActiveExpireCtx *) ctx;
     sds keystr = NULL;
     ExpireInfo info = {0};
+    ExpireAction ret = ACT_STOP_ACTIVE_EXP;
 
     /* If no more quota left for this callback, stop */
     if (activeExpireCtx->fieldsToExpireQuota == 0)
-        return ACT_STOP_ACTIVE_EXP;
+        return ret;
```

Comment:
I think the if should return explicit `ACT_STOP_ACTIVE_EXP`.
And move declaration of `ret` to be `ExpireAction ret;` before the if-else condition that actually make use of it.

Reply:
done with [`7183314` (#13329)](https://github.com/redis/redis/pull/13329/commits/71833145036834c51c51025ecd7be2aa9459363b)